packages/editor/src/components/post-card-panel/index.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	Icon,
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+	__experimentalText as Text,
+	PanelBody,
+} from '@wordpress/components';
+import { store as coreStore } from '@wordpress/core-data';
+import { useSelect } from '@wordpress/data';
+import { __, sprintf } from '@wordpress/i18n';
+import { humanTimeDiff } from '@wordpress/date';
+import { decodeEntities } from '@wordpress/html-entities';
+
+/**
+ * Internal dependencies
+ */
+import { store as editorStore } from '../../store';
+import { unlock } from '../../lock-unlock';
+
+export default function PostCardPanel( { className, actions, children } ) {
+	const { modified, title, templateInfo, icon } = useSelect( ( select ) => {
+		const {
+			getEditedPostAttribute,
+			getCurrentPostType,
+			getCurrentPostId,
+			__experimentalGetTemplateInfo,
+		} = select( editorStore );
+		const { getEditedEntityRecord } = select( coreStore );
+		const _type = getCurrentPostType();
+		const _id = getCurrentPostId();
+		let _templateInfo;
+		const _record = getEditedEntityRecord( 'postType', _type, _id );
+		return {
+			title: _templateInfo?.title || getEditedPostAttribute( 'title' ),
+			modified: getEditedPostAttribute( 'modified' ),
+			id: _id,
+			templateInfo: __experimentalGetTemplateInfo( _record ),
+			icon: unlock( select( editorStore ) ).getPostIcon(
+				_type,
+				_record?.area
+			),
+		};
+	} );
+	const description = templateInfo?.description;
+	const lastEditedText =
+		modified &&
+		sprintf(
+			// translators: %s: Human-readable time difference, e.g. "2 days ago".
+			__( 'Last edited %s' ),
+			humanTimeDiff( modified )
+		);
+
+	return (
+		<PanelBody>
+			<div
+				className={ classnames( 'editor-post-card-panel', className ) }
+			>
+				<HStack
+					spacing={ 3 }
+					className="editor-post-card-panel__header"
+				>
+					<Icon
+						className="editor-post-card-panel__icon"
+						icon={ icon }
+					/>
+					{ !! title && (
```

Comment:
This doesn't check if there is support for title, but checks if there is a title. Additionally this top part of the card has the icon and the actions. Without a title and no actions we show just an icon. If we also have actions(but no title), might affect the styles - didn't check though.

In site editor right now we show the title, so given it's tied to the rest of the `header` design we could use some design feedback.. --cc @jameskoster @jasmussen

Reply:
Hi @ntsekouras,
> This doesn't check if there is support for title, but checks if there is a title.

If there is no title support the title is empty so this check covers the lack of title support and a case where there is title support but the post has no title.
The UI for the no-title case did not look great. I changed things now, when there is no title we show "No Title" This approach was already being used in multiple cases like Post Title block etc.

Reply:
>If there is no title support the title is empty so this check covers the lack of title support

Not necessarily. For example I tested by removing the support for title for `pages`, but they already had titles from before. So some plugins could do something similar I guess 🤔 .

The fallback is fine for now I think though.